packages/catalog-search/src/config/index.js

```diff
@@ -19,4 +20,7 @@ export const features = {
   ENABlE_PATHWAYS: (
     process.env.FEATURE_ENABLE_PATHWAYS || hasFeatureFlagEnabled(FEATURE_ENABLE_PATHWAYS)
   ),
+  SUBTITLE_FACET: (
+    process.env.FEATURE_SUBTITLE_FACET || hasFeatureFlagEnabled(FEATURE_SUBTITLE_FACET)
```

Comment:
the flag is temporary ... for rollout ... right?

Reply:
@irfanuddinahmad yes.. we only want to show this dropdown on explore-catalog for now.